### PR TITLE
Updating Identity Engine > Embedded Authentication content for Node.js SDK basic sign-out use case

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Adding Basic Sign-out use case for Node.js language selector in Identity Engine > Embedded Authentication [SDK use-case](https://developer.okta.com/docs/guides/oie-embedded-sdk-use-cases/aspnet/oie-embedded-sdk-use-case-basic-sign-out/).
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-408181](https://oktainc.atlassian.net/browse/OKTA-408181)

### CODE REVIEW

**Review by shuowu-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-out/nodejs/integrationsteps.md [hunk 1]
```diff
@@ -1 +1,51 @@
...
+![Sign out link](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-out-link.png
+ "Sign out link")
+
+</div>
+
+### Step 2: Close Okta SSO session and revoke access token
+
+When the sign-out request is initiated, create the following flow:
+
+1. Obtain the access token from the active session state. For example, by calling `getAuthClient` from `getAuthClient.js` in the SDK sample application.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-out/nodejs/integrationsteps.md:18]:
```suggestion
1. Obtain the access token from the active session state. For example, obtain the access token by calling `getAuthClient` from `getAuthClient.js` in the SDK sample application.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-out/nodejs/integrationsteps.md [hunk 2]
```diff
@@ -1 +1,51 @@
...
+
+1. Call the `getSignOutRedirectURL` to close the Okta SSO session before revoking the token (`revokeAccessToken`) and invalidating the local session, as shown in the SDK sample application's `logout.js` file:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-out/nodejs/integrationsteps.md:20]:
```suggestion
1. Call the `getSignOutRedirectURL` to close the Okta SSO session before you revoke the token (`revokeAccessToken`) and invalidate the local session, as shown in the SDK sample application's `logout.js` file:
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-out/nodejs/integrationsteps.md [hunk 3]
```diff
@@ -1 +1,51 @@
...
+    req.session.destroy();
+    // Clear okta session with logout redirect
+    res.redirect(signoutRedirectUrl);
+  } catch (err) {
+    console.log('/logout error: ', err);
+  }
+});
+```
+
+>**Note**: Revoking the refresh token is optional, as it requires `offline_access` in scopes.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-out/nodejs/integrationsteps.md:42]:
```suggestion
>**Note**: Revoking the refresh token is optional, as it requires that you request the `offline_access` scope.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-out/nodejs/integrationsteps.md [hunk 4]
```diff
@@ -1 +1,51 @@
...
+
+### Step 3: Send user to the signed out page
+
+After the access token is revoked and the local app session is no longer valid, redirect the user to the signed-out page.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-out/nodejs/integrationsteps.md:46]:
```suggestion
After the access token is revoked and the local app session is no longer valid, redirect the user to the sign-out page.
```

**Review by susanharper-okta — APPROVED** [review]
just a couple nits - approving so you can merge after reviewing/committing/ignoring...haha

**Pull request merged**